index.js

```diff
@@ -67,9 +64,13 @@ var Dropzone = React.createClass({
       height: this.props.size || 100,
       borderStyle: this.state.isDragActive ? "solid" : "dashed"
     };
- 
+
+    if (this.props.className) {
```

Comment:
This one looks like a problem to me. It means only if one gives a `className` prop, will it pick the `style` prop.